lib/serverengine/multi_spawn_server.rb

```diff
@@ -19,12 +19,20 @@ module ServerEngine
 
   class MultiSpawnServer < MultiWorkerServer
     def initialize(worker_module, load_config_proc={}, &block)
-      @pm = ProcessManager.new(
-        auto_tick: false,
-        graceful_kill_signal: Daemon::Signals::GRACEFUL_STOP,
-        immediate_kill_signal: Daemon::Signals::IMMEDIATE_STOP,
```

Comment:
I understand that QUIT signal doesn't work on Windows. Does KILL signal work on Windows?
If not, `ProcessManager#tick` needs a hack like `system("taskkill /f /pid #{pid}")` when it sends KILL signal like:

```
-           Process.kill(signal, pid)
+           if ServerEngine.windows? && (signal == :KILL || signal == :SIGKILL)
+             system("taskkill /f /pid #{pid}")
+           else
+             Process.kill(signal, pid)
+           end
```